addons/point_of_sale/static/src/js/models.js

```diff
@@ -806,6 +806,30 @@ exports.PosModel = Backbone.Model.extend({
         });
     },
 
+    // load the missing partner for an order
+    load_missing_partners: function(order){
+        var self = this;
+        return new Promise(function (resolve, reject) {
+            var fields = _.find(self.models, function(model){ return model.label === 'load_partners'; }).fields;
+            var domain = [['id','=', order.partner_id[0]]];
+            self.rpc({
+                model: 'res.partner',
+                method: 'search_read',
+                args: [domain, fields],
+            }, {
+                timeout: 3000,
+                shadow: true,
+            })
+            .then(function (partners) {
+                if (self.db.add_partners(partners)) {   // check if the partners we got were real updates
+                    resolve();
+                } else {
+                    reject(new Error('Failed in updating partners.'));
```

Comment:
Not translatable I believe?

Reply:
It was used in line 803 in the same file